scenarios/authentication/rest_test.go

```diff
@@ -0,0 +1,24 @@
+package rest
+
+// Import resty into your code and refer it as `resty`.
+import (
+	"testing"
+)
+
+func TestGetUserDetails(t *testing.T) {
+
+	cred, err := NewTokensier()
+	if err != nil {
+		t.Errorf("Error : %s", err)
+	}
+	tkn, err := cred.FetchKuberaToken()
```

Comment:
cred.FetchKuberaToken().getRequest(cred.Host + "v3/account/")
// can i use like this ??